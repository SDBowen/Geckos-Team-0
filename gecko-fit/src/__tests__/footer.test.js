import React from 'react';
import ReactShallowRenderer from 'react-test-renderer/shallow';
import Footer from '../components/footer';

test('Should render Footer correctly', () => {
    const renderer = new ReactShallowRenderer();
    renderer.render(<Footer />);
    expect(renderer.getRenderOutput()).toMatchSnapshot();
});
