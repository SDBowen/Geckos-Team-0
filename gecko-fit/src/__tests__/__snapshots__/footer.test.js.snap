// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Footer correctly 1`] = `
<footer
  className="footer"
>
  <p
    className="footer__chingu-link"
  >
    Created in 
    <a
      href="https://chingu.io/"
    >
      Chingu Voyage-4
    </a>
  </p>
  <p
    className="footer__github-link"
  >
    <a
      href="https://github.com/chingu-voyage4/Geckos-Team-0"
    >
      Link to Github
    </a>
  </p>
</footer>
`;
